EIPS/eip-5450.md

```diff
@@ -58,50 +60,59 @@ In the second validation phase control-flow analysis is performed on the code.
 
 *Terminating instructions* refers to the instructions either:
 
-- ending function execution: `RETF`, or
-- ending whole EVM execution: `STOP`, `RETURN`, `REVERT`, `INVALID`.
+- ending function execution: `RETF`, `JUMPF`, or
```

Comment:
"separate EIP" again, the following ones are OK to not have it, but at least once for the Spec section is good to have

Reply:
should `RETURNCONTRACT` mention have the same then?

Reply:
oh boy, I've missed RETURNCONTRACT popping up here... my top-level opinion doesn't really change (should not mention at all), but if we mention, we should consistent in how we mention :(. We cannot leave the reader puzzled whether RETURNCONTRACT is something in legacy EVM, a typo, copy-pasta or leftovers from prev revision or whatnot.

But once more, I really think these EIPs should be as independent as possible, rather than trying to be the megaspec and cover all aspects in each thoroughly